### ISSUE TITLE
[PG-9] Placeholder not showing line by line on some browsers

### ISSUE THREAD

**Issue by CExBOommen** [CExBOommen/CExBOommen.github.io#13]
Placeholder not showing line by line on some browsers, especially some versions of chrome and not at all working on mobile.

**Comment by shardy613** [comment]
IMO - I wouldn't worry about mobile for generating picklists

**Comment by CExBOommen** [comment]
@shardy613 Yeah, Its just a reminder for myself, so i can test this issue on phone to reproduce it.

**Comment by CExBOommen** [comment]
@shardy613 Have edited the description
